library/src/extra/rocsparse_csrgeam_impl.cpp

```diff
@@ -0,0 +1,658 @@
+                                                                    beta,
+                                                                    descr_B,
+                                                                    nnz_B,
+                                                                    csr_val_B,
+                                                                    csr_row_ptr_B,
+                                                                    csr_col_ind_B,
+                                                                    descr_C,
+                                                                    csr_val_C,
+                                                                    csr_row_ptr_C,
+                                                                    csr_col_ind_C);
+        if(status != rocsparse_status_continue)
+        {
+            RETURN_IF_ROCSPARSE_ERROR(status);
+            return rocsparse_status_success;
+        }
+
+        const rocsparse_spgeam_descr descr       = nullptr;
+        void*                        temp_buffer = nullptr;
+        RETURN_IF_ROCSPARSE_ERROR(rocsparse::csrgeam_core(handle,
+                                                          descr,
+                                                          rocsparse_operation_none,
+                                                          rocsparse_operation_none,
+                                                          m,
+                                                          n,
+                                                          alpha,
+                                                          descr_A,
+                                                          nnz_A,
+                                                          csr_val_A,
+                                                          csr_row_ptr_A,
+                                                          csr_col_ind_A,
+                                                          beta,
+                                                          descr_B,
+                                                          nnz_B,
+                                                          csr_val_B,
+                                                          csr_row_ptr_B,
+                                                          csr_col_ind_B,
+                                                          descr_C,
+                                                          csr_val_C,
+                                                          csr_row_ptr_C,
+                                                          csr_col_ind_C,
+                                                          temp_buffer));
+        return rocsparse_status_success;
+    }
+}
+
+template <typename T, typename I, typename J>
+rocsparse_status rocsparse::csrgeam_template(rocsparse_handle             handle,
+                                             const rocsparse_spgeam_descr descr,
```

Comment:
From what I am seeing none of them are used

Reply:
Am I missing something? the spgeam_descr is not used in csrgeam.

Reply:
Fixed.